src/plugins/external_tool_support/src/bedtools/BedtoolsIntersectTask.cpp

```diff
@@ -179,6 +181,9 @@ QList<Task *> BedtoolsIntersectAnnotationsByEntityTask::onSubTaskFinished(Task *
         Document *resultDoc = loadResultTask->getDocument();
         CHECK_EXT(resultDoc != NULL, setError(tr("Result document is NULL")), res);
         result = resultDoc->findGObjectByType(GObjectTypes::ANNOTATION_TABLE);
+        AnnotationTableObject *ato = qobject_cast<AnnotationTableObject *>(result.first());
+        CHECK_EXT(ato != NULL, setError(tr("Failed to cast to AnnotationTableObject")), res);
+        renameAnnotationsFromBed(ato->getRootGroup());
```

Comment:
Process all annotation table objects from the list (result)?

Reply:
According to knowledge how to bedtools works there is only one annotation table in the resulting file.